### DISCUSSION TITLE
🆕 Subfolders (Public Preview)

### DISCUSSION THREAD

**Discussion by zsofiakm** [grafana/grafana#69772] [General]
## What's new in Grafana v10
Improve team onboarding and resource management with subfolders in Grafana. This new feature allows you to organize dashboards into a multi-level folder hierarchy aligned with your team's unique needs, be it business units, departments, or teams. The intuitive folder structure makes sifting through your dashboards for relevant data a more straightforward task.

_Public Preview available in v10.0 in Grafana OSS, Grafana Cloud (Free, Pro, Advanced), and Grafana Enterprise._

![image](https://github.com/grafana/grafana/assets/97596715/78526349-6745-4df6-8c20-17efbd0ff79a)

### 1. Enable the feature (Public Preview)
To get started creating subfolders, enable the `nestedFolders` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support. 
We advise switching on this feature solely on **instances designated for testing or development**, as opposed to deploying it in production environments.

### 2. Create your folder structure 
You can create folders, with nesting up to 7 levels, through Grafana’s UI or the [Folder API](https://grafana.com/docs/grafana/latest/developers/http_api/folder/).
These folders can house both dashboards and alerts. Any dashboards or alerts not placed within a specific folder will be displayed at the root level, previously known as the General folder.

### 3. Assign Permissions
You can assign users, teams or roles specific permissions at the folder level, keeping sensitive information secure.

## What's next?
The General Availability of the subfolders feature in Grafana is expected for fall 2023, and it will include enhancements based on user feedback and ongoing development, including:

- Performance enhancements
- Improved interface for creating and moving folders
- Subfolder provisioning using Terraform
- Updated documentation

Known issues:
https://github.com/grafana/grafana/issues/69775

We are keen to hear your feedback. Please feel free to share your insights and suggestions in this discussion.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as an open discussion, and we'd like those to go to our[ community forum](https://community.grafana.com/). Thank you!
